runtime/v2/vm/src/main/java/com/walmartlabs/concord/svm/UnstoppableCommand.java

```diff
@@ -0,0 +1,29 @@
+package com.walmartlabs.concord.svm;
+
+/*-
+ * *****
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2024 Walmart Inc.
+ * -----
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =====
+ */
+
+public interface UnstoppableCommand extends Command {
```

Comment:
Can we call it `FinallyBlock`, `FrameFinalizer` or just `Finalizer` ?. "Unstoppable" is too dramatic to my taste. :-)